x/dualstaking/keeper/delegator_reward.go

```diff
@@ -177,6 +193,11 @@ func (k Keeper) RewardProvidersAndDelegators(ctx sdk.Context, providerAddr sdk.A
 	contributorAddresses, contributorPart := k.specKeeper.GetContributorReward(ctx, chainID)
 	if contributorPart.GT(math.LegacyZeroDec()) {
 		contributorsNum := int64(len(contributorAddresses))
+		// Sanity check - contributorsNum > 0
+		if contributorsNum == 0 {
+			return math.ZeroInt(), math.ZeroInt(), utils.LavaFormatWarning("critical: Attempt to divide by zero", nil,
```

Comment:
same comment as last one. put what you wrote in the error field and describe the warning in the first arg

Reply:
Fixed in 20bf6f5ad2b263ea30b7641558c140ceca7a9293